micronota/bfillings/diamond.py

```diff
@@ -355,6 +355,8 @@ def parse_sam(diamond_res):
                    'qstart', 'sstart', 'evalue', 'bitscore', 'sseq']
         df = pd.DataFrame(columns=columns)
         for i, seq in enumerate(seqs):
+            if seq == []:
```

Comment:
this is wrong.  `seq` is a `Sequence` object, should nerver been a `[]`! You should change this function.